Gems/Atom/Feature/Common/Code/Source/Mesh/MeshFeatureProcessor.cpp

```diff
@@ -1114,9 +1120,18 @@ namespace AZ
                 }
                 else
                 {
-                    AZ_Warning("MeshFeatureProcessor", false, "Requested manual color as irradianceColorSource "
-                            "but I could not find an irradiance.manualColor field. Defaulting to black.");
-                    subMesh.m_irradianceColor = AZ::Color(0.0f);
+                    /* Couldn't find irradianc.manualColor -> check for an irradiance.color legacy fallback */
```

Comment:
Nitpick: It's not even a "legacy fallback", it's just another option for material types that don't support automatic irradiance color. For example, StandardMultilayerPbr as discussed above.

Reply:
```suggestion
                    // Couldn't find irradianc.manualColor -> check for an irradiance.color in case the material type 
                    // doesn't have the concept of manual vs. automatic irradiance color, allow a simpler property name
```